packaging/tests/tas-restapi/src/test/java/org/alfresco/rest/categories/CreateCategoriesTests.java

```diff
@@ -0,0 +1,223 @@
+        createdCategory.assertThat()
+                .field(FIELD_PARENT_ID).is(rootCategory.getId());
+        createdCategory.assertThat()
+                .field(FIELD_HAS_CHILDREN).is(false);
+    }
+
+    /**
+     * Check we can create several categories as children of a created category
+     */
+    @Test(groups = {TestGroup.REST_API})
+    public void testCreateSeveralSubCategories()
+    {
+        STEP("Create a category under root category (as admin)");
+        final RestCategoryModel rootCategory = new RestCategoryModel();
+        rootCategory.setId("-root-");
+        final RestCategoryModel aCategory = new RestCategoryModel();
+        aCategory.setName(RandomData.getRandomName("Category"));
+        final RestCategoryModel createdCategory = restClient.authenticateUser(dataUser.getAdminUser())
+                .withCoreAPI()
+                .usingCategory(rootCategory)
+                .createSingleCategory(aCategory);
+        restClient.assertStatusCodeIs(CREATED);
+
+        createdCategory.assertThat()
+                    .field(FIELD_NAME).is(aCategory.getName())
+                .assertThat()
+                    .field(FIELD_PARENT_ID).is(rootCategory.getId())
+                .assertThat()
+                    .field(FIELD_HAS_CHILDREN).is(false)
+                .assertThat()
+                    .field(FIELD_ID).isNotEmpty();
+
+        STEP("Create two categories under the previously created (as admin)");
+        final RestCategoryModel aSubCategory1 = new RestCategoryModel();
+        aSubCategory1.setName(RandomData.getRandomName("SubCategory"));
+        final RestCategoryModel aSubCategory2 = new RestCategoryModel();
+        aSubCategory2.setName(RandomData.getRandomName("SubCategory"));
+        final List<RestCategoryModel> categoriesToCreate = List.of(aSubCategory1, aSubCategory2);
+        final RestCategoryModelsCollection createdSubCategories = restClient.authenticateUser(dataUser.getAdminUser())
+                .withCoreAPI()
+                .usingCategory(createdCategory)
+                .createCategoriesList(categoriesToCreate);
+        restClient.assertStatusCodeIs(CREATED);
+        createdSubCategories.assertThat()
+                .entriesListCountIs(categoriesToCreate.size())
+                .getEntries()
+                    .get(0).onModel()
+                        .assertThat()
+                            .field(FIELD_NAME).is(aSubCategory1.getName())
+                        .assertThat()
+                            .field(FIELD_PARENT_ID).is(createdCategory.getId())
+                        .assertThat()
+                            .field(FIELD_HAS_CHILDREN).is(false)
+                        .assertThat()
+                            .field(FIELD_ID).isNotEmpty();
+        createdSubCategories.assertThat()
+                .entriesListCountIs(categoriesToCreate.size())
+                .getEntries()
+                    .get(1).onModel()
```

Comment:
Maybe it would be better to introduce here `for` loop against `categoriesToCreate` instead using `aSubCategory1` and `...2`.

Reply:
I'll go with a loop.